governance/votes/Executive vote - October 31, 2024.md

```diff
@@ -0,0 +1,126 @@
+- Initialize Allocator ILK for Spark Star by calling [`AllocatorInit.initIlk`](https://github.com/makerdao/dss-allocator/blob/226584d3b179d98025497815adb4ea585ea0102d/deploy/AllocatorInit.sol#L97) with its parameters set as follow:
+  - [`sharedInstance`](https://github.com/makerdao/dss-allocator/blob/226584d3b179d98025497815adb4ea585ea0102d/deploy/AllocatorInstances.sol#L19):
+    - `oracle`: [0xc7B91C401C02B73CBdF424dFaaa60950d5040dB7](https://etherscan.io/address/0xc7B91C401C02B73CBdF424dFaaa60950d5040dB7) (AllocatorOracle)
+    - `roles`: [0x9A865A710399cea85dbD9144b7a09C889e94E803](https://etherscan.io/address/0x9A865A710399cea85dbD9144b7a09C889e94E803) (AllocatorRoles)
+    - `registry`: [0xCdCFA95343DA7821fdD01dc4d0AeDA958051bB3B](https://etherscan.io/address/0xCdCFA95343DA7821fdD01dc4d0AeDA958051bB3B) (AllocatorRegistry)
+  - [`ilkInstance`](https://github.com/makerdao/dss-allocator/blob/226584d3b179d98025497815adb4ea585ea0102d/deploy/AllocatorInstances.sol#L25):
+    - `owner`:  [0xbe8e3e3618f7474f8cb1d074a26affef007e98fb](https://etherscan.io/address/0xbe8e3e3618f7474f8cb1d074a26affef007e98fb) (MCD_PAUSE_PROXY)
+    - `vault`: [0x691a6c29e9e96dd897718305427Ad5D534db16BA](https://etherscan.io/address/0x691a6c29e9e96dd897718305427Ad5D534db16BA) (AllocatorVault)
+    - `buffer`: [0xc395D150e71378B47A1b8E9de0c1a83b75a08324](https://etherscan.io/address/0xc395D150e71378B47A1b8E9de0c1a83b75a08324) (AllocatorBuffer)
+  - [`cfg`](https://github.com/makerdao/dss-allocator/blob/226584d3b179d98025497815adb4ea585ea0102d/deploy/AllocatorInit.sol#L62):
+    - `ilk`: ALLOCATOR-SPARK-A
+    - `duty`: 1000000001607468111246255079 (5.2%)
+    - `gap`: 10,000,000
+    - `maxLine`: 10,000,000
+    - `ttl`: 86,400 seconds
+    - `allocatorProxy`: [0x3300f198988e4C9C63F75dF86De36421f06af8c4](https://etherscan.io/address/0x3300f198988e4C9C63F75dF86De36421f06af8c4) (SparkProxy)
+    - `ilkRegistry`: [0x5a464c28d19848f44199d003bef5ecc87d090f87](https://etherscan.io/address/0x5a464c28d19848f44199d003bef5ecc87d090f87)
+- Whitelist [Spark ALM Proxy](https://etherscan.io/address/0x1601843c5E9bC251A3272907010AFa41Fa18347E) on the LitePSM by calling `IPSMLike(MCD_LITE_PSM_USDC_A).kiss(almProxy: 0x1601843c5E9bC251A3272907010AFa41Fa18347E)`.
+
+More information on these contracts can be found at the [dss-allocator repository](https://github.com/makerdao/dss-allocator/tree/226584d3b179d98025497815adb4ea585ea0102d); an overview of the system implementation can be found at the [README file](https://github.com/makerdao/dss-allocator/blob/226584d3b179d98025497815adb4ea585ea0102d/README.md) therein.
+
+### Add new validators for [Median](https://docs.makerdao.com/smart-contract-modules/oracle-module/median-detailed-documentation) (Medianizer)
+
+- **Authorization**: [TBC]()
+- **Proposal**: [Forum Post](https://forum.sky.money/t/spell-contents-2024-10-31/25421)
+
+If this executive proposal passes, then _all_ validators below:
```

Comment:
I am not a fan of this formatting, it reads awkwardly. Additionally, I think more resources on the Medianizer should be added.